tests/languages/css-source-code.test.js

```diff
@@ -55,9 +55,9 @@ describe("CSSSourceCode", () => {
 			});
 
 			assert.strictEqual(sourceCode.constructor.name, "CSSSourceCode");
-			assert.strictEqual(sourceCode.ast, ast);
+			assert.deepStrictEqual(sourceCode.ast, ast);
```

Comment:
Same as https://github.com/eslint/json/pull/115/files#r2158471027, I think we should keep `strictEqual` in these tests.